README.md

```diff
@@ -61,19 +59,17 @@ or
 $ npm i @mixmaxhq/promise-pool
 ```
 
-Changelog
----------
+## Changelog
```

Comment:
Should we update this? Or remove it altogether, if we'll be maintaining a CHANGELOG file?

Reply:
I think we should yes. But I don't know how to publish this package. Should I do it manually ? 
@bradvogel  do you know?, last release was 5y ago.

Reply:
Yup, the old school way :-/. This repo doesn't have semantic-release that auto-publishes.

You should be able to run `mixmax dev npm publish` (instructions [here](https://docs.google.com/document/d/1KALf-WokfvoNXxDOBruTrT6yki5hqfkj1f6OYYTlnEg/edit#bookmark=id.ht1hs081qy7p)) which runs [this script](https://github.com/mixmaxhq/mixmax/blob/78f64b927e37825f445452dd0622d7472a2b4cf0/cmds/dev-cmds/npm-cmds/scripts/publish).